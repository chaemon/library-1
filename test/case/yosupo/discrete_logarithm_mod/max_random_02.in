100
451794314 2028037 945281111
111681588 291303632 820539453
177741457 164199418 955763712
745629015 727272894 812017458
451963117 263419161 915420359
625817844 636159790 848719557
275132246 358996390 973897388
398318281 142493592 823523747
499848367 7061287 958426858
622426888 79384002 814371411
513456960 313082157 819141560
463541289 120030523 930997331
112618850 230228300 981616163
504459522 679335110 941978648
674397136 404243924 865968779
618061916 863589554 886377174
915162678 71459492 966732311
52898442 401242089 873127242
453002785 706324970 895743307
653545259 112784412 984051746
423006839 483894301 889754573
669309742 618420173 998639242
63159789 669473697 874387944
785773594 78145328 815139439
227723599 266280591 844133152
832249497 530649785 928456734
760049622 285284864 815373180
571051694 550324427 902609245
844757180 27279851 972354129
711708323 241389523 830518775
95282135 388198169 864720706
122021473 420267989 889837203
207534299 532166635 864714249
660803648 200677819 866284175
741997440 706435381 918884818
578541378 175415879 866950056
628486949 510176861 883961528
926002845 368100055 977621669
621658024 257571540 861809585
864636173 167265452 921236313
497916596 550847494 930572986
758608226 568056529 989674283
860690631 433527490 880906003
438782170 532703055 886561732
464996346 459050659 862487177
448906708 277614808 927734486
626274617 597214909 847750211
180426754 43412191 992584545
542275936 209267734 863671486
386859145 84098970 920714502
161552223 119634118 845864784
809928697 130763529 830107399
582647216 556636623 812892742
123862180 846344645 885915696
46147265 639857947 960906495
431937569 294403014 999250212
121038459 919321994 952536706
721753845 474964155 846195566
550621781 433177566 850311966
949027832 568805827 984574801
969391498 910191630 995162271
474202086 64884460 914548429
935590236 364830411 990773306
688217587 560792314 815994905
152225730 99227632 975410063
59551364 587481860 955212024
942223509 362895767 952926330
322778310 715453291 897223158
327962270 434672391 814371406
375024120 531433894 905983829
528901509 808264285 877198629
745140973 56268139 991560546
111933668 204658092 878552171
764401593 732123932 828006754
672712383 468924696 951422227
700743486 608345215 869577874
180352086 336664642 809765191
168859884 576353231 811699495
304003094 83056178 841641163
98992432 243035791 839345059
819145862 701455874 858164836
72726621 87302527 937122019
582435991 6608577 951591524
175084790 683266391 995587148
484718004 617253574 962107255
495045358 527208105 819761681
478112150 284212393 817361672
937789476 305802577 974758283
338281497 114673431 868539774
389190500 108514522 801698158
78431292 878717496 999552880
793108760 281288305 934564625
885567777 339577893 923292565
159610396 205321258 888841466
104693224 424187352 904936434
211866651 180200905 875820195
726140430 867745814 876730017
208630878 271708931 837730737
816371134 747264388 886394331
90765576 209724128 971206644
