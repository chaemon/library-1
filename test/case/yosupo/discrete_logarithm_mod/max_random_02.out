486533240
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
109604413
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
157344379
-1
-1
-1
-1
-1
-1
339865374
-1
-1
77839498
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
3208642
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
159327522
-1
175472984
-1
35751637
-1
-1
190658018
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
274947673
-1
-1
-1
-1
-1
-1
234927
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
